node/smeshing_service_test.go

```diff
@@ -38,6 +77,9 @@ func TestNewSmeshingService(t *testing.T) {
 		service, err := NewSmeshingService(cfg, zaptest.NewLogger(t))
 		require.NoError(t, err)
 		require.NotNil(t, service)
+		defer func() {
+			service.Close(context.Background())
+		}()
```

Comment:
NIT: why the anonymous function?

```suggestion
		defer service.Close(context.Background())
```

Reply:
IDK :D fixed, thx